app/src/main/res/values/strings.xml

```diff
@@ -1,22 +1,17 @@
 <resources>
     <string name="app_name">Goose Eye</string>
     <string name="button_back_text">Back</string>
-    <string name="pg_text">13+</string>
-    <string name="name_text">Avengers:\nEnd Game</string>
+    <string name="pg_text">+</string>
+    <string name="name_avengers">Avengers:\nEnd Game</string>
     <string name="tag_line_text">Action, Adventure, Fantasy</string>
-    <string name="review_text">125 REVIEWS</string>
+    <string name="review_text">REVIEWS</string>
     <string name="story_line_text">Storyline</string>
     <string name="story_line_details_text">After the devastating events of Avengers: Infinity War,
         the universe is in ruins. With the help of remaining allies, the Avengers assemble once
```

Comment:
It is better not to store mock text in strings. You can use `tools:text="@tools:sample/lorem/random"`

Reply:
I know how convenient is it to test layout with mock paragraphs! Maybe it is better not to store mock text in strings?Would you mind if we replace it with `tools:text="@tools:sample/lorem/random"`?